stable/storageoscluster-operator/templates/NOTES.txt

```diff
@@ -32,5 +32,10 @@ metadata:
 spec:
   secretRefName: storageos-api
   secretRefNamespace: default
+  images:
+    nodeContainer: "storageos/node:1.1.2" # StorageOS version
```

Comment:
Instead of hard-coding and updating this every time, we can use {{.Chart.AppVersion}}.
https://helm.sh/docs/chart_template_guide/ has some examples of built in objects.

Reply:
I don't think Chart.AppVersion is what we are looking for. The value for that is currently 0.0.8. Can we create a new Chart.StorageOS version or something alike?

Reply:
Actually, `{{ Chart.appVersion }}` seems to be the correct thing to do here .. 
see: https://github.com/helm/helm/blob/master/docs/charts.md#the-chartyaml-file

Reply:
It seems that `appVersion` is already used to identify the operator version: "appVersion:1.1.0" - from: https://github.com/storageos/charts/pull/51/files?short_path=04c6e90#diff-f088d92bfd16284c11a3c605bda42914R2

Reply:
This PR is no longer needed. We decided to not add another manual change for every release.